fbw-a32nx/src/systems/instruments/src/PFD/FMA.tsx

```diff
@@ -925,17 +938,19 @@ class B2Cell extends DisplayComponent<CellProps> {
                 text2 = '';
             }
 
+            // CLB armed should be centered, other modes are aligned to the left on newer DMCs
+            this.xposSub.set(clbArmed ? 50.801795 : 40.777474);
```

Comment:
Sorry for the late review, but I feel like this could be handled easier by just adding spaces to the text for CLB

Reply:
Good idea, I didn't think of that when I first made the change. I've changed it to use spaces and looks fine too. 
![image](https://github.com/flybywiresim/aircraft/assets/119708186/2164f453-7943-4040-bcd7-4ac67d552298)